Lib/fontmake/font_project.py

```diff
@@ -306,9 +308,10 @@ def build_interpolatable_otfs(self, designspace, **kwargs):
         """
         return self._build_interpolatable_masters(designspace, ttf=False, **kwargs)
 
-    def build_variable_font(
+    def build_variable_fonts(
         self,
-        designspace,
+        designspace: designspaceLib.DesignSpaceDocument,
+        variable_fonts=True,
```

Comment:
the type of this should be `str` and should default to `".*"` regex pattern (build all VFs regardless of the name); a bool value made sense for the instances parameter which one can set to False; setting `variable_fonts=False` in a method named `build_variable_fonts` doesn't make much sense.